gneiss/stats/composition.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from skbio import DistanceMatrix
+
+
+def variation_matrix(X):
+    """ Calculate Aitchison variation matrix.
+
+    Parameters
+    ----------
+    X : np.array
+        Contingency table where the samples are rows and the features
+        are columns.
+    Returns
+    -------
+    np.array
+        Total variation matrix
+    """
+    v = np.zeros((X.shape[1], X.shape[1]))
+    for i in range(X.shape[1]):
```

Comment:
Just added a reference: V. Pawlowsky-Glahn, J. J. Egozcue, R. Tolosana-Delgado (2015), Modeling and Analysis of Compositional Data, Wiley, Chichester, UK